packages/components/input-otp/src/use-input-otp.ts

```diff
@@ -230,6 +230,7 @@ export function useInputOtp(originalProps: UseInputOtpProps) {
         ref: inputRef,
         name: name,
         value: value,
+        autoFocus: originalProps.autoFocus,
```

Comment:
why we need to extract from `originalProps` instead of just using `autoFocus` in L107?

Reply:
Done in: fffa4bbc2b6baba16f51b970c49d439833b988f3